geode-docs/tools_modules/gfsh/command-pages/backup.html.md.erb

```diff
@@ -59,6 +59,11 @@ backup disk-store --dir=value [--baseline-dir=value]
 <td>Directory that contains the baseline backup used for comparison during an incremental backup.
 <p>An incremental backup operation backs up any data that is not present in the directory specified in <span class="keyword parmname">&#8209;&#8209;baseline-dir</span>. If the member cannot find previously backed up data or if the previously backed up data is corrupt, the command performs a full backup on that member.</p></td>
 </tr>
+<tr>
+<td><span class="keyword parmname">&#8209;&#8209;include-disk-stores</span></td>
+<td>List of disk-stores to include in backup.
+<p>Selective backup of disk-stores listed in <span class="keyword parmname">&#8209;&#8209;include-disk-stores</span>. Specified disk-stores must exist in system. If parameter is not specified, all disk-stores are backup.</p></td>
```

Comment:
I suggest the following changes to the phrasing:

The specified disk-stores must exist in the system.

If this parameter is not specified, all disk-stores are included in the backup.

Reply:
Thanks for comments. Updated.

Reply:
Any reason for not adding "this" in  "If this parameter is not specified..."?

Reply:
I skipped that part. I will update.